src/core/types/models.ts

```diff
@@ -65,7 +65,5 @@ export interface ModelRegistry {
 }
 
 export interface ContextMetrics {
-  tokensUsed: number;
   tokensRemaining: number;
-  percentageRemaining: number;
```

Comment:
Since we have `tokensRemaining` in local storage and max tokens on the model config, we can calculate percentage remaining when we need to, rather than adding to conversation history in local storage.